.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+# Pull Request informations
+
+## Rebase & Merge default requirements
+
+1. Green build for master branch
+2. Wait 24 hours after PR creation
+3. Green job for PR
+4. Approved PR
+
+## PR labels default process
+
+- READY_FOR_REVIEW  &rightarrow;  REVIEW_DONE  &rightarrow;  READY_FOR_MERGE
+
+## Tests
+
+- [ ] Are there **Unit tests**?
+- [ ] Are there **Integration tests**?
+- [ ] Do we need a new **UI test**?
+
+## PR workflow progress
+
+1. [ ] Tagged with relevant **PR labels**
+2. [ ] Green **job for PR**
+3. [ ] PR was created more then **24 hours ago**
```

Comment:
rules is in fact more precisely:
- created more than 24 hours ago or all committers approved it